src/main/java/com/cleanroommc/groovyscript/compat/mods/ModSupport.java

```diff
@@ -179,7 +178,11 @@ public static void init() {
         for (GroovyContainer<?> container : containerList) {
             if (container.isLoaded()) {
                 container.onCompatLoaded(container);
-                container.get().initialize();
+                container.get().initialize(container);
+                int i = 0;
+                for (String s : container.getAliases()) {
+                    ExpansionHelper.mixinConstProperty(ModSupport.class, s, container.get(), i++ > 0);
+                }
```

Comment:
pretty sure that what entry is visible and which are hidden can change based on iteration order?
would something like this work:
```suggestion
                ExpansionHelper.mixinConstProperty(ModSupport.class, container.getModId(), container.get(), false);
                for (String s : container.getAliases()) {
                    ExpansionHelper.mixinConstProperty(ModSupport.class, s, container.get(), true);
                }
```